src/api/api.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const API = import.meta.env.VITE_API_URL;
+
+export async function getData({
+  page = 1,
+  pageSize = 10,
+  orderBy = "recent",
+  keyword = "",
+}) {
+  const query = `page=${page}&pageSize=${pageSize}&orderBy=${orderBy}${
+    keyword ? `&keyword=${keyword}` : ""
+  }`;
+  const res = await axios.get(
+    `${API}products?${query}`
+  );
```

Comment:
💬 여담
아래처럼 간단하게도 작성 가능합니다~

```suggestion
  const query = { page, pageSize, orderBy, ...(keyword && { keyword }) };
  const res = await axios.get(`${API}products`, { params: query });
```